http/oidc/src/test/java/org/wildfly/security/http/oidc/KeycloakConfiguration.java

```diff
@@ -33,20 +59,28 @@
 
 import io.restassured.RestAssured;
 
+import static org.wildfly.security.http.oidc.Oidc.*;
+import static org.wildfly.security.http.oidc.OidcBaseTest.TEST_REALM;
+
 /**
  * Keycloak configuration for testing.
  *
  * @author <a href="mailto:fjuma@redhat.com">Farah Juma</a>
  */
 public class KeycloakConfiguration {
 
+    private static String algorithmType;
     private static final String USER_ROLE = "user";
     private static final String ADMIN_ROLE = "admin";
     public static final String ALICE = "alice";
     public static final String ALICE_PASSWORD = "alice123+";
     private static final String BOB = "bob";
     private static final String BOB_PASSWORD = "bob123+";
     public static final String ALLOWED_ORIGIN = "http://somehost";
+    public static final boolean EMAIL_VERIFIED = false;
+    public static final String KEYSTORE_PATH = "/home/ppaul/jwt.keystore";
+    public static final String EC_KEYSTORE_PATH = "/home/ppaul/jwtEC.keystore";
```

Comment:
I am not sure where to add these. These files are created during runtime. But creating it the target folder under http/oidc did not work for me. Maybe I was specifying the paths wrong. But this will need to be changed for the final version to a target folder or something.